internal/services/oracledatabase/exadata_infrastructure_resource_test.go

```diff
@@ -0,0 +1,211 @@
+// Copyright © 2024, Oracle and/or its affiliates. All rights reserved
+
+package oracledatabase_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2024-06-01/cloudexadatainfrastructures"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracledatabase"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type ExadataInfraResource struct{}
+
+func (a ExadataInfraResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := cloudexadatainfrastructures.ParseCloudExadataInfrastructureID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.OracleDatabase.OracleDatabaseClient.CloudExadataInfrastructures.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving Exadata Infrastructure %s: %+v", id, err)
```

Comment:
The string formatter for the IDs contain this information so we don't need to specify it in the error message
```suggestion
		return nil, fmt.Errorf("retrieving %s: %+v", id, err)
```

Reply:
Done.